stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,113 @@
+# Stubs for multiprocessing.context
+
+# NOTE: These are incomplete!
```

Comment:
Are they? Looks like we're missing a few exception classes and other small things.